kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/convert/BigQueryRecordConverter.java

```diff
@@ -80,6 +80,23 @@ public BigQueryRecordConverter(boolean shouldConvertDoubleSpecial) {
     return convertStruct(kafkaConnectRecord.value(), kafkaConnectSchema);
   }
 
+  /**
+   * Convert a {@link SinkRecord} key into the String Representation
+   *
+   * @param kafkaConnectRecord The Kafka Connect record to convert. Key Must be of type {@link Struct},
```

Comment:
Please update the comment with the reason we need to force the key type to be STRUCT